scorep/__main__.py

```diff
@@ -11,6 +12,39 @@ def _err_exit(msg):
     sys.exit(1)
 
 
+def print_help():
+    print("""
+Usage: python -m scorep [options] [--] your_program.py [args]
+
+Score-P Python instrumentation wrapper. The following options control how the program is instrumented and executed:
+
+  --help                   Show this help message and exit.
+  --mpi                    Enable MPI instrumentation (equivalent to --mpp=mpi).
+  --keep-files             Keep temporary files after execution.
+  --verbose, -v            Enable verbose output for debugging and tracing.
+  --nocompiler             Disable compiler-based instrumentation.
+  --nopython               Disable instrumentation of Python code.
+                           Instrumentation can still be enabled later from within the application's source code.
+  --noinstrumenter         Same as --nopython.
+  --instrumenter-type=<t>  Specify custom instrumenter type (e.g., cProfile).
+  --instrumenter-file=<f>  Path to a Python script that is executed before the application.
```

Comment:
if `<t>` and `<f`> are not referenced in the help text, they should be self-explanatory, i.e., not abbreviated

```suggestion
  --instrumenter-type=<type>
                           Specify custom instrumenter type (e.g., cProfile).
  --instrumenter-file=<file>
                           Path to a Python script that is executed before the application.
```

or

```suggestion
  --instrumenter-type=<t>  Specify custom instrumenter type (`<t>` can `cProfile` or `…`).
  --instrumenter-file=<f>  Execute Python script `<f>` before the application.
```